Andrea / exercise-1/index.html

```diff
@@ -0,0 +1,181 @@
+            <p><span class="title-span">read<br>more</span> open your eyes</p>
+        </div>
+        
+        <div class="body">
+            <!-- <section class="columns-format"> quitar esta clase y ponerla en body -->
+                <div class="body-img">
+                    <img src="side_img.jpg" alt="imagen">
+                </div>
+                <div class="body-content">
+                    <h3>why you should read more</h3>
+                    <p>
+                        Lorem ipsum dolor, sit amet consectetur adipisicing elit. 
+                        Cupiditate, voluptatum ut assumenda doloremque, velit dolor
+                        ab nemo placeat aspernatur quam, voluptatibus nulla explicabo veritatis deleniti amet tempore! Blanditiis, nihil qui.
+                    </p>
+                    
+                    <h3>finding books you like</h3>
+                    <p>
+                      Lorem ipsum dolor, sit amet consectetur adipisicing elit. 
+                      Cupiditate, voluptatum ut assumenda doloremque, velit dolor
+                      ab nemo placeat aspernatur quam, voluptatibus nulla explicabo ver
+                    </p>
+                </div>
+            <!-- </section> -->
+          
+        <aside class="recommendations">
+            <h4>recommendations</h4>
+            <ul>
+                <li>
+                    <img src="icon.png" alt="icon">
+                    <p>the name of the wind <span>5/5</span></p>
+                </li>
+                <li>
+                    <img src="icon.png" alt="icon">
+                    <p>shadowhunters <span>3.5/5</span></p>
+                </li>
+                <li>
+                    <img src="icon.png"" alt="icon">
+                    <p>the song of achilles <span>5/5</span></p>
+                </li>
+                <li>
+                    <img src="icon.png" alt="icon">
+                    <p>a little life <span> 5/5</span></p>
+                </li>
+                <li>
+                    <img src="icon.png" alt="icon">
+                    <p>pride and prejudice<span> 5/5</psan></p>
+                </li>
+            </ul>
+        </aside>
+    </div>
+    </main>
+
+    <footer>
+        <nav class="footer-menu">
+            <img class="logo" src="logo.png" alt="image">
+            <ul class="footer-menu__links">
+                <li>
+                    <a href="#">travel</a>
+                </li>
+                <li>
+                    <a href="#">explore</a>
+                </li>
+                <li>
+                    <a href="#">works</a>
+                </li>
+                <li>
+                    <a href="#">about us</a>
+                </li>
+                <li>
+                    <a href="#">contacts</a>
+                </li>
+                <!-- Botones -->
+                <li>
+                    <input class="footer-menu__input" type="text" placeholder="Your email">
```

Comment:
inputs must be inside a form element, and this form should be a sibling of the UL. Also add the correct type to the input as seen here https://developer.mozilla.org/en-US/docs/Web/HTML/Element/input/email